api/include/opentelemetry/context/threadlocal_context.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include "opentelemetry/context/runtime_context.h"
```

Comment:
Also include context.h directly, rather than relying on getting it from runtime_context.h

Reply:
Included directly.